addons/website_sale_collect/static/src/tests/tours/buy_with_click_and_collect.js

```diff
@@ -4,15 +4,21 @@ import * as tourUtils from '@website_sale/js/tours/tour_utils';
 
 registry.category('web_tour.tours').add('website_sale_collect_buy_product', {
     url: '/shop',
-    checkDelay: 50,
     steps: () => [
         ...tourUtils.searchProduct("Test CAC Product"),
-        clickOnElement("Test Product", 'a:contains("Test CAC Product")'),
+        clickOnElement("Test Product", '.oe_product a:contains("Test CAC Product")'),
         clickOnElement("Open Location selector", '.o_click_and_collect_availability'),
-        clickOnElement("Choose location", '#submit_location_large'),
+        clickOnElement("Choose location", "#submit_location_large:contains(choose this location)"),
```

Comment:
Could you please leave the single quotes? We use single for technical strings and double for functional ones.
Why do we need to add `contains` ?
```suggestion
        clickOnElement("Choose location", '#submit_location_large:contains(choose this location)'),
```